src/dndbeyond/base/monster.js

```diff
@@ -829,15 +829,15 @@ class Monster extends CharacterBase {
         let hp = null;
         let max_hp = null;
         let temp_hp = null;
-        const groups = $(".ct-creature-pane .ct-collapsible__content .ct-creature-pane__adjuster-group,.ct-creature-pane .ddbc-collapsible__content .ct-creature-pane__adjuster-group");
+        const groups = $(".b20-creature-pane .ct-collapsible__content .b20-creature-pane__adjuster-group,.b20-creature-pane .ddbc-collapsible__content .b20-creature-pane__adjuster-group");
```

Comment:
The change to `.b20-creature-pane` makes sense, since you add that class, but the others wouldn't work.
There's no `.b20-creature-pane__adjuster-group` but I do see a `styles_adjusterGroup_*` which we should be using.
![{29C459FC-D902-49F1-AAD8-E749D15B0083}](https://github.com/user-attachments/assets/287e57f1-4d4b-450c-a317-c0737388f541)

Reply:
Yeah i did not mean to change that one it was a find and replace mistake, i should be more careful. 
yeah you are right tho